src/Examine/Examine.ruleset

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what is this doing? IIRC any previous rules were based on autogenerated rules from 0.x when medium trust was a requirement